README.md

```diff
@@ -140,6 +141,13 @@ You can run build in watch mode (useful for real-time development with e.g Dashb
 pnpm watch
 ```
 
+To push to remote, you need to install dependencies in the legacy project, from root folder:
```

Comment:
**Suggestion:** I would rather add this step in line 129 when you install dependencies.

So it will look like this:
```sh
pnpm install
cd legacy && npm install
```

Reply:
I think we should require installing dependencies for both `legacy` and `src` to develop MacawUI locally - as in effect pushing to remote will work out the box